src/main/java/frc/robot/subsystems/AddressableLEDSubsystem.java

```diff
@@ -97,20 +113,43 @@ public void humanColorMethod(ColorType colorType) {
     else if(ColorType.PURPLE == colorType)
     {
       //Sets each LED to Purple
-      for(int i = DRIVER_START_RANGE; i < LEDBuffer.getLength(); i++)
-      {
+      for(int i = DRIVER_START_RANGE; i < LEDBuffer.getLength(); i++) {
         LEDBuffer.setHSV(i, PURPLE_H, PURPLE_S, PURPLE_V);
       }
       LED.setData(LEDBuffer);
     }
   }
 
+  /**
+   * show a color on the led panel on the front of the robot
+   * @param colorType purple or yellow
+   * @return
+   */
   public CommandBase HumanColor(ColorType colorType) {
-    return runEnd(() -> humanColorMethod(colorType), this::LEDOff);   
+    // subsystem requirement is PURPOSEFULLY OMITTED
+    return new RunCommand(
+      // turn leds on while command is active
+      () -> humanColorMethod(colorType)
+    ).andThen(
+      // turn leds off when command ends
+      this::humanPlayerOff
+    );   
   }
 
+  /**
+   * show a color on the led panel on the back of the robot
+   * @param colorType purple, yellow, or red
+   * @return 
+   */
   public CommandBase DriverColor(ColorType colorType) {
-    return runEnd(() -> driverColorMethod(colorType), this::LEDOff);
+    // subsystem requirement is PURPOSEFULLY OMITTED
+    return new RunCommand(
+      // turn leds on while command is active
+      () -> driverColorMethod(colorType)
+    ).andThen(
+      // turn leds off when command ends
+      this::driverOff
+    );   
```

Comment:
convert to a `FunctionalCommand`, where run is turning the leds on and end is turning them off instead of using an `andThen`